scripts/release/inline-resources.js

```diff
@@ -93,7 +93,9 @@ function inlineTemplate(content, urlResolver) {
     const templateContent = fs.readFileSync(templateFile, 'utf-8');
     const shortenedTemplate = templateContent
       .replace(/([\n\r]\s*)+/gm, ' ')
```

Comment:
Yeah, sure. I considered reusing htmlmin, but that seemed like overkill to me haha. I'll close this if you'd prefer to take on a different approach.